TTS/trainer.py

```diff
@@ -1052,8 +1099,21 @@ def process_args(args, config=None):
         tb_logger = TensorboardLogger(experiment_path, model_name=config.model)
         # write model desc to tensorboard
         tb_logger.tb_add_text("model-config", f"<pre>{config.to_json()}</pre>", 0)
+
+        wandb_project_name = config.model
+        if config.wandb_project_name:
+            wandb_project_name = config.wandb_project_name
+
+        wandb_logger = WandbLogger(
+            disabled=config.wandb_disabled,
+            project=wandb_project_name,
+            name=config.run_name,
+            config=config,
+            entity=config.wandb_entity,
+        )
+
     c_logger = ConsoleLogger()
-    return config, experiment_path, audio_path, c_logger, tb_logger
+    return config, experiment_path, audio_path, c_logger, tb_logger, wandb_logger
```

Comment:
It doesn't need to return wandb_logger additionally since I don't think using TB and WDB together is not a common case. Instead we can ask which dashboard that they want to use and just return that. 

Something like 

```python
# in config
log_dashboard: str = "tensorboard" or "wanddb""

# in process_args
if config.log_dashboard == "tensorboard":
    dashboard_logger = TensorboradLogger()
elif config.log_dash....
    dashboard_logger = WandDBLogger()
```

Does it make sense?

Reply:
Yes it makes sense. I'll make the changes